.license-header

```diff
@@ -1,7 +1,7 @@
 // This file is part of radicle-link
 // <https://github.com/radicle-dev/radicle-link>
 //
-// Copyright (C) 2019-2020 The Radicle Team <dev@radicle.xyz>
+// Copyright (C) 2019-2020 The Radicle Foundation <hello@radicle.foundation>
```

Comment:
I believe we're minimising what needs to go in the header. @angedupre said she would get back to me on a shorter copy for it.